community-operators/okd-windows-machine-config-operator/0.0.1/okd-windows-machine-config-operator.0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,229 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: "OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/openshift-windows/okd-windows-machine-config-operator:4.6-1d646e8
+    createdAt: 2020-09-01T11:59:59Z
+    description: An operator which allows for Windows workloads to be run within OKD
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: windows-machine-config-operator
```

Comment:
For cluster monitoring to work I think the namespace should conform to `openshift-` on OCP. Is there a similar requirement for OKD?

Reply:
none that I know of, but IMO it'd make sense to keep them the same in both release variants.

Reply:
actually, all of the rules that apply to OCP will apply to OKD as well. I think prefixing with `openshift-` is best here.